packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -0,0 +1,324 @@
+import {Defer} from '@webex/common';
+
+import LoggerProxy from '../common/logs/logger-proxy';
+import {ClusterNode} from './request';
+
+import {ICE_GATHERING_STATE, CONNECTION_STATE} from '../constants';
+
+const DEFAULT_TIMEOUT = 3000;
+const VIDEO_MESH_TIMEOUT = 1000;
+
+// result for a specific transport protocol (like udp or tcp)
+export type TransportResult = {
+  result: 'reachable' | 'unreachable' | 'not tested';
```

Comment:
Might be good for 'not tested' to be changed to `untested` - no spaces? nitpick obviously but sometimes spaces cause problems

Reply:
done